src/translations/en.json

```
@@ -3060,6 +3058,7 @@
           "editor": {
             "caption": "Add Credential",
             "description": "OAuth is used to grant Home Assistant access to information on other websites without giving a passwords. This mechanism is used by companies such as Spotify, Google, Withings, Microsoft, and Twitter.",
+            "missing_credentials": "Setting up {integration} requires configuring application credentials.",
```

Can we make sure we link to the domain documentation of the integration with instructions? 
```suggestion
            "missing_credentials": "Setting up {integration} requires configuring application credentials. [See the documentation for more information.]({url})",
```